app/create/page.tsx

```diff
@@ -1,117 +1,135 @@
 "use client";
-
-import { Button, Stack, TextField, Autocomplete } from "@mui/material";
+import React from "react";
+import { Button, Stack } from "@mui/material";
 import Link from "next/link";
+import { useForm, FormProvider } from "react-hook-form";
+import FormInput from "@/components/FormInput";
+import FormAutocomplete from "@/components/FormAutocomplete";
+import * as yup from "yup";
+import { validationSchema } from "@/validation/validationSchema";
+import { yupResolver } from "@hookform/resolvers/yup";
+import {
+  OwnedOptions,
+  StatusOptions,
+  TypesOfGamesOptions,
+  PlatformOptions,
+} from "@/components/types";
+import {
+  typesOfGamesOptions,
+  ownedOptions,
+  statusOptions,
+  platfromOptions,
+} from "@/validation/options";
+import { GameFormValues } from "@/validation/options";
 
-export default function Create() {
-	function createGame(event) {
-		event.preventDefault();
+export default function Create(props) {
+  const methods = useForm<GameFormValues>({
+    defaultValues: {
+      game_name: "",
+      type_of_game: TypesOfGamesOptions.RPG,
+      owned: OwnedOptions.No,
+      status: StatusOptions.PLANING,
+      platform: PlatformOptions.STEAM,
+    },
+    resolver: yupResolver(validationSchema),
+    shouldFocusError: true,
+    criteriaMode: "all",
+    reValidateMode: "onChange",
+  });
 
-		const data = {};
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+  } = methods;
 
-		for (const [name, value] of new FormData(event.target)) {
-			data[name as string] = value;
-		}
+  const errorMsg = errors["game_name"]?.message;
 
-		fetch("api/games", {
-			method: "POST",
-			body: JSON.stringify(data),
-			headers: {
-				"Content-type": "application/json; charset=UTF-8",
-			},
-		}).then(async (response) => await console.log(response.json()));
-	}
+  function createGame(data) {
+    fetch("api/games", {
+      method: "POST",
+      body: JSON.stringify(data),
+      headers: {
+        "Content-type": "application/json; charset=UTF-8",
+      },
+    }).then(async (response) => await console.log(response.json()));
```

Comment:
Common pattern after creating an entry is to redirect user to listing view, I guess it would be better than console.log